hack/common.sh

```diff
@@ -56,6 +56,7 @@ readonly OPENSHIFT_BINARY_SYMLINKS=(
   openshift-sti-build
   openshift-docker-build
   openshift-gitserver
+  openshift-diagnostics
```

Comment:
It tab-completes though and so is discoverable :)

I agree the name is too long but I didn't really like any of the alternatives and didn't hear a consensus. I figure that might develop after people actually use it.